src/pages/Swap/index.tsx

```diff
@@ -159,7 +161,8 @@ export default function Swap({ className }: { className?: string }) {
   const loadedUrlParams = useDefaultsFromURLSearch()
   const [newSwapQuoteNeedsLogging, setNewSwapQuoteNeedsLogging] = useState(true)
   const [fetchingSwapQuoteStartTime, setFetchingSwapQuoteStartTime] = useState<Date | undefined>()
-  const swapWidgetEnabled = useSwapWidgetEnabled()
+  const swapWidgetManualOverrideEnabled = useSwapWidgetEnabled()
```

Comment:
I would prefer to see statsig integrated into the featureFlags/ logic, so that manual overrides don't need special-cased logic going forward.

The base hook (for featureFlag) could delegate to statSig, and then _this_ file would just call to `useSwapWidgetEnabled`.

Reply:
thoughts on the update?